test/dummy/db/migrate/20110208155312_set_up_test_tables.rb

```diff
@@ -114,6 +114,23 @@ def up
     end
     add_index :banana_versions, [:item_type, :item_id]
 
+    # oranges use a uuid column (string)
+    create_table :oranges, primary_key: "uuid", id: :string, force: true do |t|
+      t.string :name
+      t.timestamps null: true
+    end
+
+    # and orange_versions stores the orange uuid in item_id, a string
+    create_table :orange_versions, force: true do |t|
+      t.string   :item_type, null: false
+      t.string   :item_id,   null: false
+      t.string   :event,     null: false
+      t.string   :whodunnit
+      t.text     :object
+      t.datetime :created_at
+    end
+    add_index :orange_versions, [:item_type, :item_id]
```

Comment:
this is largely modeled after banana, without the Kitchen namespace, and changed to use a string item_id